### ISSUE TITLE
PointRend cannot run for semantic segmentation when there's only one foreground class

### ISSUE THREAD

**Issue by cmis91** [facebookresearch/detectron2#2297]
in Project/PointRend/point_rend/semantic_seg.py
Line 32:  top2_scores = torch.topk(sem_seg_logits, k=2, dim=1)[0]
when there's only one foreground class like the ballon example, I got the following error: 
“ invalid argument 5: k not in range for dimension at /pytorch/aten/src/THC/generic/THCTensorTopK.cu:26” 
I guess in semantic segmentation, I should set cfg.MODEL.SEM_SEG_HEAD.NUM_CLASSES = 2 to include the background class. Am I right?

**Comment by ppwwyyxx** [comment]
If you need help to solve an unexpected issue you observed, please include details following the ["Unexpected behaviors" issue template](https://github.com/facebookresearch/detectron2/issues/new/choose).

**Comment by cmis91** [comment]
I guess semantic segmentation is set up differently than instance segmentation. In the semantic segmentation, the number of classes has to include the background class. Actually, every pixel in the bitmask needs to have a label. The problem is solved after I changed cfg.MODEL.SEM_SEG_HEAD.NUM_CLASSES from 1 to 2.
